scripts/create_full_environment.sh

```diff
@@ -34,11 +34,12 @@ scripts/install_environment.sh
 echo "Done installing"
 
 echo "Installing kind"
-scripts/hub-cluster/kind/kind.sh install
+make bring_assisted_service
+assisted-service/hack/kind/kind.sh install
 echo "Done installing kind"
 
 echo "Installing minikube"
-scripts/hub-cluster/minikube.sh install
+assisted-service/hack/minikube/minikube.sh install
```

Comment:
we are installing both kind and minikube ?

Reply:
because we want to support both, and the deployment choice is happening during deployment stage using `DEPLOY_TARGET`, while the installations are happening before during setup